### PR TITLE
[Resources] Service Proxy

### PR DESCRIPTION
**Pull request by debo19**
Signed-off-by: Debopriya Bhattacharjee <debopriyabhattacharjee@Debopriyas-MacBook-Air.local>

**Description**
Added new article.
This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 2ec95ae7bef4fd1f3ce8f6970a795ef5e1fbea55 at: https://6212af3a5ee3974bb600da69--layer5ng.netlify.app

**Comment by debo19** [comment]
@leecalcote good to merge?

File: src/collections/resources/how-to-write-webassembly-filters-for-envoy/learn-how-to-write-wasm-filters.mdx [hunk 1]
```diff
@@ -21,7 +21,7 @@ import { ResourcesWrapper } from "../Resources.style.js";
 <ResourcesWrapper>
   <div className="intro">
     <p>Learn more about WebAssembly's use within service mesh data planes in
-      <Link className="blog" to="/learn/service-mesh-books/the-enterprise-path-to-service-mesh-architectures-2nd-edition">The Enterprise Path to Service Mesh Archictures (2nd Edition)</Link> -  free book and excellent resource for anyone looking to understand WASM filters, Lua scripts, and other options available for extending the data plane.</p>
+      <Link className="blog" to="/learn/service-mesh-books/the-enterprise-path-to-service-mesh-architectures-2nd-edition"> The Enterprise Path to Service Mesh Archictures (2nd Edition)</Link> -  free book and excellent resource for anyone looking to understand WASM filters, Lua scripts, and other options available for extending the data plane.</p>
```
Comment (leecalcote) [inline src/collections/resources/how-to-write-webassembly-filters-for-envoy/learn-how-to-write-wasm-filters.mdx:24]: Is this whitespace intentional?

**Review by leecalcote — APPROVED** [review]
Other than the question about the whitespace, yes, this is good to go.

File: src/collections/resources/how-to-write-webassembly-filters-for-envoy/learn-how-to-write-wasm-filters.mdx (hunk 1 above)
Comment (debo19) [reply src/collections/resources/how-to-write-webassembly-filters-for-envoy/learn-how-to-write-wasm-filters.mdx:24]: Yes it is, the link is missing a whitespace in front.

Comment (leecalcote) [reply src/collections/resources/how-to-write-webassembly-filters-for-envoy/learn-how-to-write-wasm-filters.mdx:24]: @debo19 i don't understand. Normally, we don't want leading whitespace to be hyperlinked. What's the need here?

Comment (debo19) [reply src/collections/resources/how-to-write-webassembly-filters-for-envoy/learn-how-to-write-wasm-filters.mdx:24]: @leecalcote oh okay I got what you mean. I was just fixing this. Working on another article PR, will add the space before the `<Link>` tag there.

Screenshot 2022-02-22 at 6 40 38 PM

**Pull request merged**